security-oauth2/src/main/java/io/micronaut/security/oauth2/client/clientcredentials/propagation/ClientCredentialsHttpClientFilter.java

```diff
@@ -76,7 +76,7 @@ public ClientCredentialsHttpClientFilter(OutgoingHttpRequestProcessor outgoingHt
                                              BeanContext beanContext) {
         this.outgoingHttpRequestProcessor = outgoingHttpRequestProcessor;
         this.oauthClientConfigurationCollection = oauthClientConfigurationStream
-                .filter(config -> config.getClientCredentials().map(ClientCredentialsConfiguration::isEnabled).orElse(false))
```

Comment:
why this change?

Reply:
That's an artifact of when I started looking into deprecating the method referred to by #1091. I didn't mean to push that change. I have reverted that for now.